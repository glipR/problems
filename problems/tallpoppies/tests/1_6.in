6 31 73
279705727 586083506 777784991 634965519 482807431 652364817 805678239 613017050 246996246 124277841 816024268 436305659 590249566 760535821 295390456 568836879 922613072 668154020 626569879 933642908 166892650 973832026 778574880 667946609 462392543 793537855 948144923 279373396 881055288 648454738 136124286
10576518 0
25715969 0
30324844 1 701920179
64504953 0
66619944 1 2136129
79717041 0
81740397 1 759288742
104534431 0
154331954 1 804800499
162025112 0
174367043 0
184008093 0
191826337 0
216769191 1 736796811
218580853 0
222558500 1 438380734
222720446 1 572065402
228275357 1 349378021
243035740 0
259377777 0
297507201 1 88599722
301394545 1 380299059
310943978 1 178651505
317924009 0
329862308 0
349727740 1 856124899
360370653 0
365172688 1 422179262
388486571 0
393904001 0
416549314 0
431626450 0
463692908 1 622255872
486098864 0
510976610 1 167719662
518644890 0
519268943 1 43648815
533370600 1 444193498
535935681 1 425726684
539512356 0
554830134 1 28068312
559741036 0
588149795 1 461106129
588425892 1 241331411
617184057 0
629918315 0
642942201 1 673243634
653017640 1 692134247
662568216 1 373924148
665399201 0
668764109 0
680768784 1 227469730
688822002 1 966457100
713076581 1 59776569
728432484 1 578883970
734663913 0
749053028 0
757545836 0
771375942 0
783212066 1 719910919
844567062 0
845048977 1 410459103
849327895 1 331002327
867678338 1 860092907
877669572 0
880470173 1 914754354
921546107 1 576592316
922036703 0
933323756 1 345352764
937910443 1 97087492
960427163 0
981275104 1 221135973
985544798 1 316600441
