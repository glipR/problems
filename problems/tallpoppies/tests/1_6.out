1037291134
1102440737
1320672626
1400915318
1508261874
1788174940
1851880497
1897586413
1929532902
2089270109
2218750261
2224420682
2575490663
2631538665
2810678656
2965884945
2989993885
3112312934
3180008266
3502676690
3680706219
3757045251
3841253647
4165496885
4215815549
4287785662
4292290381
4687356874
4763164528
4792271262
4795148302
5203526658
5390295273
5411028993
5364979443
